src/guide/scaling-up/testing.md

```diff
@@ -137,19 +137,13 @@ const { getByText } = render(Stepper, {
     max: 1
   }
 })
-
-getByText('0') // Afirmación implícita de que "0" está dentro del componente
-
+getByText('0') // Aserción implícita de que "0" está dentro del componente.
 const button = getByText('increment')
-
-// Enviar un evento de clic a nuestro botón de incremento.
+// Ejecuta un evento de clic al botón de incremento.
 await fireEvent.click(button)
-
 getByText('1')
-
 await fireEvent.click(button)
 ```
```

Comment:
A space is missing between the lines: 
<img width="636" alt="Screenshot 2023-12-18 at 20 30 22" src="https://github.com/icarusgk/vuejs-spanish-docs/assets/38413630/6f9e9cef-9a79-444e-a37c-a17e13d861c8">

https://github.com/vuejs/docs/commit/acac67f5fdde6cad7982bbb376faf6a4940be580#diff-5caeaf6ca264c1f52cc58e010b0d62fc5be0c1599a53e5b6fb81d80c32d03e8cR141-R153